docs/generated/packages/next/executors/build.json

```diff
@@ -72,6 +72,11 @@
       "experimentalAppOnly": {
         "type": "boolean",
         "description": "Only build 'app' routes"
+      },
+      "experimentalBuildMode": {
+        "type": "string",
+        "description": "Change the build mode.",
+        "enum": ["compile", "generate"]
```

Comment:
There is some confusion w.r.t. to the "default" option. The documentation shows it as available, if you pass it the CLI will complain though.